graphene_django/rest_framework/mutation.py

```diff
@@ -124,8 +126,13 @@ def __init_subclass_with_meta__(
     def get_serializer_kwargs(cls, root, info, **input):
         lookup_field = cls._meta.lookup_field
         model_class = cls._meta.model_class
-
         if model_class:
+            # check if input is Enum, and serialize value into input
+            for k, v in input.items():
+                attr = getattr(cls.Input, k)
+                type_ = getattr(attr, 'type')
```

Comment:
There's still a pre-commit black formatting issue here which is failing CI. `'type'` -> `"type"`

Reply:
Updated.